src/WebJobs.Script/Description/CSharp/FunctionEntryPointAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    [System.AttributeUsage(AttributeTargets.Method, Inherited = false, AllowMultiple = false)]
+    public sealed class FunctionEntryPointAttribute : Attribute
```

Comment:
This was added after some conversations with the PMs about the resolution behavior. I agree that it adds complexity to support something that is possibly unnecessary, so I'm fine removing it.